src/test/integrationSuiteSetup.ts

```diff
@@ -0,0 +1,5 @@
+import { db } from '../database';
+
+afterAll(async () => {
+  await db.close();
+});
```

Comment:
I am still trying to find where this fits in, but frankly I have not followed closely how any of the other pieces fit together, so I don't intend to spend too long looking.

Reply:
Found it above.

Reply:
I can add a comment explaining what is going on for the casual observer (as you saw below, this is Jest Magic and it's not explicit).

This file is also where I'd add the automatic "db up / down before each integration test" logic.